config/dev.exs

```diff
@@ -0,0 +1 @@
+
```

Comment:
can revert

Reply:
Unfortunately `mix format --check-formatted` didn't like the empty file without the newline, hence the change.

Reply:
ah of course